Modules/IO/ImageBase/include/itkConvertPixelBuffer.h

```diff
@@ -46,6 +46,12 @@ template <typename InputPixelType, typename OutputPixelType, typename OutputConv
 class ITK_TEMPLATE_EXPORT ConvertPixelBuffer
 {
 public:
+  /** Constructor */
```

Comment:
For completeness:

```c++
    // Remove all constructor/ destructor / assignments for class with only static member functions.
    ConvertPixelBuffer(const ConvertPixelBuffer&) = delete;
    ConvertPixelBuffer(ConvertPixelBuffer&&) = delete;
    ConvertPixelBuffer& operator=(const ConvertPixelBuffer&) = delete;
    ConvertPixelBuffer& operator=(ConvertPixelBuffer&&) = delete;
    virtual ~ ConvertPixelBuffer() = delete;
```

Reply:
@hjmjohnson The only other class containing static member functions is `Core/Common/itkFloatingPointExceptions.h`, otherwise the rest of the fixes have been made.

Reply:
Please change the itkFloatingPointExceptions.h to also add a comment and explicitly delete all constructors/destructors/ assignment operators.